preliz/distributions/discrete_weibull.py

```diff
@@ -107,6 +107,14 @@ def skewness(self):
     def kurtosis(self):
         return num_kurtosis(self)
 
+    def mode(self):
```

Comment:
optimization routines should be inside optimization module, as they can be potentially reused. The 100 is a magic number, maybe a better option is to use self.ppf(0.9999) or similar. I tried with a couple of examples and the mode is not always right,